README.md

```diff
@@ -49,13 +82,15 @@ In order to install stackrox-central-services you will need a secure password. T
 ```sh
 openssl rand -base64 20 | tr -d '/=+' > stackrox-admin-password.txt
 ```
-
 From here you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
 ```sh
 helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="$(cat stackrox-admin-password.txt)"
 ```
 
-If you're deploying StackRox on a small node like a local development cluster, run the following command to reduce StackRox resource requirements. Keep in mind that these reduced resource settings are not suited for a production setup.
+#### Limited Central Resource Installation
```

Comment:
->`Limited Resource Central Installation`

Reply:
We wanted to document how to downsize the resource requirements if you deploy on small, edge, or single-cluster setups. Maybe something like "Installing Central in clusters with limited resources" would be more adequate?

Reply:
It is updated to provide further context and differentiate the installation better.